compile_stdlib.py

```diff
@@ -0,0 +1,140 @@
+#!/usr/bin/env python3
+
+"""
+Convert ouroborous modules to JavaScripted pyc files that we can load.
+"""
+
+# TODO: we should just import ourborous and generate the pycs from there
+
+import base64
+import glob
+import os
+import os.path
+import py_compile
+import sys
+import tempfile
+
+enabled_modules = [
+    'colorsys',
+    'copyreg',
+    'this',
+]
```

Comment:
I think that is something we could easily modify it for. Is that something we want to hold this PR on?